feature(doc_auto_cfg) has been removed

I guess this code has to be adjusted?

https://github.com/smol-rs/futures-lite/blob/70c589f372064613c36a987f16d878b58755bfd6/src/lib.rs#L37

Error from the docs.rs build server https://docs.rs/crate/glycin/3.0.4/builds/2602719

```
[INFO] [stderr] error[E0557]: feature has been removed
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/index.crates.io-1949cf8c6b5b557f/futures-lite-2.6.1/src/lib.rs:37:29
[INFO] [stderr]    |
[INFO] [stderr] 37 | #![cfg_attr(docsrs, feature(doc_auto_cfg))]
[INFO] [stderr]    |                             ^^^^^^^^^^^^ feature has been removed
[INFO] [stderr]    |
[INFO] [stderr]    = note: removed in CURRENT_RUSTC_VERSION; see <https://github.com/rust-lang/rust/pull/138907> for more information
[INFO] [stderr]    = note: merged into `doc_cfg`
[INFO] [stderr] 
[INFO] [stderr] For more information about this error, try `rustc --explain E0557`.
[INFO] [stderr] error: could not compile `futures-lite` (lib) due to 1 previous error
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -34,7 +34,7 @@
 #![doc(
     html_logo_url = "https://raw.githubusercontent.com/smol-rs/smol/master/assets/images/logo_fullsize_transparent.png"
 )]
-#![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
 #![forbid(unsafe_code)]
 
 #[cfg(feature = "alloc")]

